src/Layers/TiledMapLayer.js

```diff
@@ -149,6 +150,22 @@ export var TiledMapLayer = TileLayer.extend({
     TileLayer.prototype.onAdd.call(this, map);
   },
 
+  _checkSRSupport: function (map, spatialReference) {
+    if (map.options.crs === L.CRS.EPSG3857 && (spatialReference === 102100 || spatialReference === 3857)) {
+      // web mercator tile services are supported by leaflet
+      return true;
+    } else if (map.options.crs === L.CRS.EPSG4326 && (spatialReference === 4326)) {
+      // wgs84 tile services are supported by leaflet
+      return true;
+    } else if (map.options.crs && map.options.crs.code && (map.options.crs.code.indexOf(spatialReference) > -1)) {
+      // using proj4 and defining a custom crs enables support for tile services published in other projections
+      return true;
```

Comment:
Hey, today I found the time to test out the newest version of `esri-leaflet` and sadly it did not work out as I hoped for.
Before the changes things related to `proj4` would go into the `else` block and show a warning if `proj4` was missing (which caused the original issue I described here https://github.com/Esri/esri-leaflet/issues/1019).
After the changes things related to `proj4` goes into the main block and fires `lodmap` event.
Because of this, I only get the initial map loaded and when I do zooming or dragging around the map no other base map tiles are loaded or even requested to be loaded.

Here is the issue visible on Plunker https://plnkr.co/edit/kFneHCA8CBMpAbg12mwp?p=preview
And if you just change the version to `esri-leaflet@2.1.2` in the `index.html` you would see that it works fine.

Reply:
ugh. totally my fault. fix is in #1051.

Reply:
Heh, no problem, thanks for the fast fix 👍